primitives/staking/src/offence.rs

```diff
@@ -79,6 +102,14 @@ pub trait Offence<Offender> {
 	/// number. Note that for GRANDPA the round number is reset each epoch.
 	fn time_slot(&self) -> Self::TimeSlot;
 
+	/// Whether this offense needs to disable offenders until the next era starts.
+	///
+	/// In previous versions any slashed offenders got disabled in every offence. Now each offence
```

Comment:
This comment should go into the pr description to make it easier to migrate. In the code it can be removed.